### PR TITLE
Hit the api

### PR DESCRIPTION
**Pull request by jerryowusu**
## Hit the Api

# In this milestone I:

- Created a new game using the API with the help of the leaderBoard API documentation
- Updated the refresh button to receive data from the API and parse it from JSON
- Updated the form submit button to send data to the API
- Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/modules/game.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  };
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  })
+    .then((res) => res.json());
+};
```
Comment (Kingstalux) [inline src/modules/game.js]: - [x] Great job so far 👍 your App displays well and works as expected but in the project requirements you are to use `async` and `await` JavaScript features to consume the API but you have implemented your features using `promises`. Kindly rectify this issue in all the cases where you are fetching data. Below is an implemented example:
- Async function:
Annotation 2022-03-02 161448

File: src/modules/get.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import renderList from './display.js';
+
+const getData = () => fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/jplay/scores/')
+  .then((response) => response.json())
+  .then((data) => renderList(data.result));
```
Comment (Kingstalux) [inline src/modules/get.js]: - [x] Same issue here, you should use `async` and `await` 👍

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @jerryowusu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

149637855-7722d04c-b45d-4185-925f-7de047106461


## Highlights
✔️ Linters are passing
✔️ Great job using JavaScript arrow functions
✔️ Good commit messages

## Required Changes ♻️
_Check the comments under the review._
- [ ] (OPTIONAL) It would be better to remove the live demo section from your README since it currently serves no purpose now so as to make your README file more professional 😄 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/game.js (hunk 1 above)
Comment (jerryowusu) [reply src/modules/game.js]: Thanks for alerting me. It just escaped me. Resolved now

File: src/modules/get.js (hunk 2 above)
Comment (jerryowusu) [reply src/modules/get.js]: Thanks for alerting me. It just escaped me. Resolved now

**Review by moise10r — APPROVED** [review]
### Hi @jerryowusu , 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/game.js (hunk 1 above)
Comment (moise10r) [reply src/modules/game.js]: Issue well addressed and Well done :+1:

**Comment by jerryowusu** [comment]
Thanks bro

**Pull request merged**